example_config_usage.py

```
@@ -0,0 +1,167 @@
+    # 获取包含环境变量的配置
+    llm_config = config.get_module_config("llm")
+    api_key = llm_config["clients"]["openai"]["api_key"]
+    
+    # 掩码显示 API 密钥（只显示最后4个字符）
+    if api_key and len(api_key) > 4:
+        masked_key = "••••" + api_key[-4:]
+    else:
+        masked_key = "[REDACTED]"
+    
+    print(f"🔑 API Key: {masked_key}")
+    print(f"📝 如果设置了 OPENAI_API_KEY 环境变量，会被替换")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)